com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -125,23 +125,17 @@ public static Collection<Platform[]> data() {
 
     private File getOutputDirFile(String outputDir, String projectName) {
         String folderName = projectName;
-        switch (platform)
+        if (platform == Platform.Arm64MacOS || platform == Platform.X86_64MacOS ||
+            platform == Platform.Arm64Ios || platform == Platform.X86_64Ios)
```

Comment:
The big downside of turning the enum into a class is that it cannot be used as a constant.

A little annoying, but I feel I'd like to have the platforms plugin-able to keep them truly separate.

In the future, I think we'll have an IPlatform.java interface, that give us details needed (like shader language information etc.)